uhabits-core/src/jvmMain/java/org/isoron/uhabits/core/ui/screens/habits/show/views/HistoryCard.kt

```diff
@@ -56,38 +57,54 @@ class HistoryCardPresenter(
     val screen: Screen,
 ) : OnDateClickedListener {
 
-    override fun onDateClicked(date: LocalDate) {
+    override fun onDateClicked(date: LocalDate, isLongClick: Boolean) {
```

Comment:
Instead of adding a boolean to the method argument, could you please create two methods: `onDateShortPress` and `onDateLongPress`?